tests/vm/basevm.py

```diff
@@ -59,19 +84,25 @@ class BaseVM(object):
     poweroff = "poweroff"
     # enable IPv6 networking
     ipv6 = True
-    def __init__(self, debug=False, vcpus=None):
+    def __init__(self, debug=False, vcpus=None, config={}):
```

Comment:
They may prefer passing config=None
I remember that python folks does not like passing em[ty classes

Reply:
OK, I will switch it to none.